pycqed/analysis/measurement_analysis.py

```diff
@@ -124,9 +154,10 @@ def save_fig(self, fig, figname=None, xlabel='x', ylabel='y',
                     print('WARNING: Could not set tight layout')
             try:
                 fig.savefig(
```

Comment:
Perhaps we can re-set the set_xlabel and set_ylabel in the savefig to work around #226, as this is also where the rescaling happens (from 'tight')

Reply:
I will not add this feature to this PR. The save_fig method is used by all the analysis classes to save all sorts of different types of figures (with or without subplots, TwoD and not TwoD plots, many different axis labels and units). Hence, if I understand this correctly, one has to think about how to make the X/Y labels resetting flexible enough (set_xlabel/set_ylabel need to know the axis, the label, and the unit), and how to add these changes to save_fig such that they do not break existing code. I'm afraid that changing this method here will break routines in PR #347.